File: apps/backend/src/FinanceManager.Domain/Entities/ContaReceber.cs
using System;

namespace FinanceManager.Domain.Entities
{
    public class ContaReceber
    {
        public Guid Id { get; set; }
        public Guid UserId { get; set; }
        public string Descricao { get; set; } = default!;
        public decimal Valor { get; set; }
        public DateTime DataRecebimento { get; set; }
        public string Status { get; set; } = default!;  
        public string Categoria { get; set; } = default!;
        public DateTime CreatedAt { get; set; }
    }
}

File: apps/frontend/src/types/ContaReceber.ts
export interface CreateContaReceberRequest {
    userId: string;
    descricao: string;
    valor: number;
    dataRecebimento: string; // ISO date
    categoria: string;
}

export interface UpdateContaReceberRequest {
    id: string;
    descricao: string;
    valor: number;
    dataRecebimento: string; // ISO date
    categoria: string;
    status: 'Pendente' | 'Recebido';
}

export interface ContaReceberDto {
    id: string;
    descricao: string;
    valor: number;
    dataRecebimento: string;
    categoria: string;
    status: 'Pendente' | 'Recebido';
    createdAt: string;
}
